tests/test.sh

```diff
@@ -1,3 +1,61 @@
-#!/bin/sh
+#!/bin/bash
 
-echo "Not implemented, so it passed"
+cd tests
+
+source ./testutil.sh
+
+message "Running synbiohub test suite."
+message "Cleaning old test containers if they exist"
+
+bash ./testcleanup.sh
+
+
+
+# Clone the necessary repositories
+message "pulling mehersam/SBOLTestRunner"
+if cd SBOLTestRunner; then
+    git pull;
+    cd ..;
+else
+    git clone --recurse-submodules https://github.com/mehersam/SBOLTestRunner;
+fi
+
+
+
+#message pulling mhersam/SBOLTestRunner
+#if cd SynBioHubRunner
+#git clone --recurse-submodules https://github.com/mehersam/SynBioHubRunner
+
+message "pulling synbiohub/synbiohub-docker"
+if cd synbiohub-docker; then
+    git pull;
+    cd ..;
+else
+    # clone the synbiohub docker compose file in order to run docker containers
+    git clone https://github.com/synbiohub/synbiohub-docker;
+fi
+
+message "Building SBOLTestRunner"
+cd SBOLTestRunner
+mvn package
+cd ..
+
+message "Starting SynBioHub from Containers"
+docker-compose -f ./synbiohub-docker/docker-compose.yml -p testproject up &
```

Comment:
Instead of running this in the background with `&`, run with the `-d` option, which puts the containers in the background after they're built.